src/common/Eval.res

```diff
@@ -0,0 +1,70 @@
+type evalResult = result<string, string>
+
+module Config = {
+  type fromWorker = ResultMessage({forCode: string, result: evalResult})
+  type fromApp = EvalMessage(string)
+  let make = () => Worker.make("./EvalWorker.mjs")
+}
+
+module EvalWorker = Worker.Make(Config)
+
+let worker = EvalWorker.make()
```

Comment:
Opted for a module-level singleton here as tracking this in React is fiddly and not really needed (I don't think we need to support multiple concurrent evaluations with multiple workers)

Reply:
Looks like next crashes (in dev at least) when constructing a worker at the top level. We now create a worker per hook invocation.